content/docs/studio/view-settings.md

```diff
@@ -0,0 +1,92 @@
+# Additional Settings for a View
+
+If you are creating a view for a DVC repo, and if the DVC repo is at the root of
+the Git repository and does not reference remote/cloud storage, then you can
+successfully visualize it without specifying additional settings.
+
+In other cases, you could be trying to create views from non-DVC repositories.
+Or, your project may be in a sub-directory inside a monorepo. Or, the data to be
+visualized could be in custom files in your repository or even in remote / cloud
+storages. In all these scenarios, you will need to specify advanced settings so
+that DVC Studio is able to access the data required for visualization. These
+scenarios have been explained in details below.
+
+## Non-DVC repositories
+
+DVC Studio creates views by identifying datasets, metrics and hyperparameters
```

Comment:
```suggestion
DVC Studio creates Views by identifying datasets, metrics and hyperparameters
```

Reply:
Not sure if the plural 'Views' is also a keyword for us. I think non-cap views would be ok. What do you think?